src/assets/components/Single-Article.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react"
+import { useParams } from 'react-router-dom';
+import NewArticleCard from "./New-Article-Card";
+import { getArticleById } from "../../api";
+import { format } from 'date-fns';
+import "./css/new-article-card.css";
+
+
+export const SingleArticle = ({isLoading, setIsLoading}) => {
+    const { article_id } = useParams()
+
+    const [individualArticle, setIndividualArticle] = useState([]) 
+
+     useEffect(() => {
+     getArticleById(article_id).then((articleData) => {
+         setIndividualArticle(articleData)
+         setIsLoading(false)
+    })
+ }, [article_id])
+
+ {isLoading ? (
+    <p>Loading...</p>
+  ) : false}
+  
+    return(
+<main id="single-article">
+        <ul className="articles-list">
+          {individualArticle.map(
+            ({
+              author,
+              title,
+              article_id,
+              article_img_url,
+              comment_count,
+              created_at,
+              topic,
+              votes
+            }) => (
+              <NewArticleCard
```

Comment:
instead of mapping through the articles on the list to display them on each individual article page, you should be making a separate call to the api to get the individual article